exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -338,3 +338,142 @@ drill.exec: {
 drill.jdbc: {
   batch_queue_throttling_threshold: 100
 }
+
+
```

Comment:
Perhaps add a bit more info here:
"Defaults for all system and session options. Provided here for easy customization in Drill distributions in the drill-distrib.conf file. Note: users should not set these values in their drill-override.conf file; users should use ALTER SYSTEM and ALTER SESSION to set the options."